src/Integration.UnitTests/Telemetry/TelemetryDataTests.cs

```diff
@@ -55,6 +55,12 @@ public void XmlSerialization_RoundTrips()
                 ShowHotspot = new ShowHotspot
                 {
                     NumberOfRequests = 567
+                },
+
+                Taint = new Taint
+                {
+                    NumberOfOpenedOnServer = 88,
```

Comment:
Minor: this is one name that will definitely have to change as it doesn't make sense grammatically.

Reply:
IEnglishSpeakNot